w3-admin.md

```diff
@@ -147,3 +148,25 @@ export const get = capability({
   },
 })
 ```
+
+### `cid/get`
+
+Get information about a CID. This does not return the actual data identified by the CID, just metadata our
+system tracks, e.g. the spaces the content identified by a given CID has been uploaded to and the
```

Comment:
Hmm, to check my understanding, this is a list of spaces a shard has been stored in and/or registered as an upload in?

Maybe this is a bit too generic as `cid/get`, I'd be tempted to call it something like `shard/stat`, `shard/info` or `shard/get`.

Or are we talking about a `root` CID? How does that work with `store/add` (which only deals with shard CIDs)?

Either way this is confusing and I think needs some clarification.